activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/parser/handler/ServiceTaskParseHandler.java

```diff
@@ -75,7 +75,7 @@ protected void executeParse(BpmnParse bpmnParse, ServiceTask serviceTask) {
       serviceTask.setBehavior(webServiceActivityBehavior);
       
     } else {
-      logger.warn("One of the attributes 'class', 'delegateExpression', 'type', 'operation', or 'expression' is mandatory on serviceTask " + serviceTask.getId());
+      serviceTask.setBehavior(bpmnParse.getActivityBehaviorFactory().createDefaultServiceTaskBehavior(serviceTask));
```

Comment:
that might cause a lot of info being logged IMO, maybe debug?

Reply:
@Salaboy I agree with @balsarori: this would cause a lot of info being logged. In addition, previously there were a check during the validation phase triggering an error if the information was missing. So I'm not convinced about logging this information here.

Reply:
@erdemedeiros debug maybe is the best one.. for me it will make some sense to be able to check when the default behavior was selected.

Reply:
done.